app/serializers/api/declaration_serializer.rb

```diff
@@ -0,0 +1,44 @@
+module API
```

Comment:
Noticed some differences in the order of some of the fields if we compare with the same endpoint in ECF app (I know we removed some fields as those apply to ECT/Mentors only). Should we follow the same order? Just trying to start a chat on this.. tks

Reply:
I think we should order reasonably in a way that makes the most sense, they don't have to match, with the timestamps hopefully at the end :)

Reply:
I've pushed the timestamp to the end, as much as it pains my OCD 😅 making the other fields match ECF would result in multiple view blocks which I think would be 🤮